<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#ddd</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Decimal Length:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBoxDecimal">
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Background-color:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxBgColor">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>white</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>orange</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>pink</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>purple</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>lime</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dodgerblue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>limegreen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>darkred</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Font-color:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxFtColor">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>white</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>orange</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>pink</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>purple</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>lime</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dodgerblue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>limegreen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>darkred</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonSave">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReset">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
